.evergreen/ocsp/certs.yml

```diff
@@ -49,9 +49,31 @@ certs:
     keyUsage: [digitalSignature, keyEncipherment]
     extendedKeyUsage: [serverAuth, clientAuth]
 
-- name: 'server-mustStaple.pem'
```

Comment:
This change isn't clear to me. Aren't drivers relying on a `server-mustStaple.pem` file to exist?

Reply:
Ah thanks for the catch! I had mutated this definition instead of creating a new one.  I've restored the old definition (the corresponding pem file was never changed)